app/league/templates/dashboard/dashboard.html

```diff
@@ -19,7 +19,7 @@ <h2 class="sub-header">Players</h2>
           <tbody>
             {% for player in players -%}
             <tr>
-              <td>{{ player.aga_id }}</td>
+                <td><a href="players/{{player.id}}"> {{ player.aga_id }}</a></td>
```

Comment:
Made the whole name linked. Whole row would require js. Maybe revisit later, and handle all the tables consistently.